Gruntfile.js

```diff
@@ -1,63 +1,187 @@
-'use strict';
+var request = require('superagent');
 
 module.exports = function(grunt) {
+  'use strict';
+
+  require('load-grunt-tasks')(grunt);
+
   grunt.initConfig({
-    'git-describe': {
-      options: {
+    pkg: grunt.file.readJSON('package.json'),
+    changelog: 'CHANGELOG.md',
+
+    'gh-release': {},
 
-      },
-      'run': {}
-    },
     closurecompiler: {
       minify: {
         requiresConfig: 'git-revision',
         files: {
-          "gen/axs_testing.js": [
-              "./lib/closure-library/closure/goog/base.js",
-              "./src/js/axs.js",
-              "./src/js/BrowserUtils.js",
-              "./src/js/Constants.js",
-              "./src/js/AccessibilityUtils.js",
-              "./src/js/Properties.js",
-              "./src/js/AuditRule.js",
-              "./src/js/AuditRules.js",
-              "./src/js/AuditResults.js",
-              "./src/js/Audit.js",
-              "./src/audits/*"
+          '.tmp/build/axs_testing.js': [
+              './lib/closure-library/closure/goog/base.js',
+              './src/js/axs.js',
+              './src/js/BrowserUtils.js',
+              './src/js/Constants.js',
+              './src/js/AccessibilityUtils.js',
+              './src/js/Properties.js',
+              './src/js/AuditRule.js',
+              './src/js/AuditRules.js',
+              './src/js/AuditResults.js',
+              './src/js/Audit.js',
+              './src/audits/*'
           ]
         },
         options: {
-          "language_in": "ECMASCRIPT5",
-          "formatting": "PRETTY_PRINT",
-          "summary_detail_level": 3,
-          "warning_level": "VERBOSE",
-          "compilation_level": "SIMPLE_OPTIMIZATIONS",
-          "output_wrapper": "<%= grunt.file.read('scripts/output_wrapper.txt') %>",
-          "externs": "./src/js/externs/externs.js"
+          'language_in': 'ECMASCRIPT5',
+          'formatting': 'PRETTY_PRINT',
+          'summary_detail_level': 3,
+          'warning_level': 'VERBOSE',
+          'compilation_level': 'SIMPLE_OPTIMIZATIONS',
+          'output_wrapper': "<%= grunt.file.read('scripts/output_wrapper.txt') %>",
+          'externs': './src/js/externs/externs.js'
         }
       }
     },
+
     qunit: {
       all: ['test/index.html']
+    },
+
+    copy: {
+      dist: {
+        expand: true,
+        cwd: '.tmp/build',
+        src: '**/*',
+        dest: 'dist/js'
+      }
+    },
+
+    clean: {
+      all: ['.tmp', 'dist']
+    },
+
+    bump: {
+      options: {
+        files: ['package.json', 'bower.json'],
+        updateConfigs: ['pkg'],
+        pushTo: "<%= grunt.config.get('gh-release.remote') %>",
+        commitFiles: ['package.json', grunt.config.get('changelog'), 'bower.json', 'dist']
+      }
+    },
+
+    prompt: {
+      'gh-release': {
```

Comment:
I'm mistaken, github shows all tags in the releases tabs *only if* there are explicit releases defined.